rel/overlay/etc/vm.args

```diff
@@ -60,3 +60,35 @@
 
 # Set maximum SSL session lifetime to reap terminated replication readers
 -ssl session_lifetime 300
+
+## TLS Distribution
+## Use TLS for connections between Erlang cluster members.
+## http://erlang.org/doc/apps/ssl/ssl_distribution.html
+##
+## Generate Cert(PEM) File
+## This is just an example code to generate a certfile (PEM).
+## This is not an endorsement of specific expiration limits, key sizes, or algorithms.
+##    $ openssl req -newkey rsa:2048 -new -nodes -x509 -days 3650 -keyout key.pem -out cert.pem
+##    $ cat key.pem cert.pem > dev/server.pem && rm key.pem cert.pem
+##
+## Config File Example
```

Comment:
Add a link perhaps to Erlang TLS dist setup so users can add other options like mutual TLS if they want to https://erlang.org/doc/apps/ssl/ssl_distribution.html